client/js/table-model.js

```diff
@@ -1,8 +1,14 @@
 class TableModel {
-  constructor(numCols=10, numRows=20) {
+  constructor(numCols=4, numRows=5) {
     this.numCols = numCols;
     this.numRows = numRows;
     this.data = {};
+    for (let row = 0; row < numRows - 1; row +=1) {
```

Comment:
It's unconventional to have for loops in a class constructor. Is there a different way you can achieve the same functionality?